### PR TITLE
Translate into traditional Chinese(繁體中文).

### PR DESCRIPTION
**Pull request by amazingandyyy**
- Translate into traditional Chinese 翻譯成繁體中文
- add Andy to supporter list.

### CODE REVIEW

**Comment by amazingandyyy** [comment]
@1995parham I translated into traditional Chinese(繁體中文).

File: README-TW.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+你可以在[此处](https://github.com/mohebifar/made-in-iran)查看完整列表。
+
+## 如何支持我們
+
+你可以將這則訊息傳不在任何地方、並且“和支持最基本的言論自由一樣“維護自由軟體。
+你可以：
+
+- 星星並在分享此專案到社交平台，加上 #githubForEveryone 這個 hashtag。
+- 在 “together” 文件節裡新增以你的用戶名命名的資料夾。
```
Comment (iyotetsuya) [inline README-TW.md]: `文件節` I think this is a typo?

File: README-TW.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- [@sepandhaghighi](https://github.com/sepandhaghighi)/[**pycm**](https://github.com/sepandhaghighi/pycm)
+- [@sepandhaghighi](https://github.com/sepandhaghighi)/[**art**](https://github.com/sepandhaghighi/art)
+- [@imaNNeoFighT](https://github.com/imaNNeoFighT)/[**fl_chart**](https://github.com/imaNNeoFighT/fl_chart)
+- [@nuxt](https://github.com/nuxt)/[**nuxt.js**](https://github.com/nuxt/nuxt.js)
+- [@bootstrap-vue](https://github.com/bootstrap-vue)/[**bootstrap-vue**](https://github.com/bootstrap-vue/bootstrap-vue)
+- [@wolfengine](https://github.com/wolfengine)/[**wolf.engine**](https://github.com/wolfengine/wolf.engine)
+- [@soroushchehresa](https://github.com/soroushchehresa)/[**vue-soundcloud**](https://github.com/soroushchehresa/vue-soundcloud)
+- [@soroushchehresa](https://github.com/soroushchehresa)/[**unsplash-wallpapers**](https://github.com/soroushchehresa/unsplash-wallpapers)
+
+你可以在[此处](https://github.com/mohebifar/made-in-iran)查看完整列表。
```
Comment (iyotetsuya) [inline README-TW.md]: [IMO] `此处` -> `此處`

File: README-TW.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## 如何支持我們
+
+你可以將這則訊息傳不在任何地方、並且“和支持最基本的言論自由一樣“維護自由軟體。
```
Comment (iyotetsuya) [inline README-TW.md]: `你可以將這則訊息傳不在任何地方` -> `你可以將這則訊息傳到任何地方`

File: README-TW.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+1. GitHub 突然限制了我们使用私有專案的權限，而我們希望 GitHub 最起碼要在採取該限制措施時先警告/告知我們。
+
+2. GitHub 甚至採取更進一步措施，限制任何只要曾經使用過來自伊朗的IP的用戶使用 Github 。 但 Github 其實可以只要限制現在訪問的IP位置，就像其他美國公司採取的措施。 現在的政策根本就是國籍歧視。
+
+最後，我們希望 GitHub 團隊：
+
+- 允許我們下載我們的私有專案和 gist ，並且能在權限設置設定成公開。
+  - **更新**：現在可以公開私有專案了、但還是不能將私有 gist 改成公開。
+- 改變現有的限制政策，不可根據用戶國籍來評斷限制與否。
+- 公開宣布他們得決定並且將為這次限制事件公開道歉（就像之前 Slack 公司為類似情況做的道歉：https://slackhq.com/an-apology-and-an-update ）。
```
Comment (iyotetsuya) [inline README-TW.md]: `公開宣布他們得決定並且將為這次限制事件公開道歉` this sounds not fluency to me, but not sure how

File: README-TW.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+![alt text](./message.png)
+
+~~**提示:** 你可以使用[此擴充套件](https://github.com/JafarAkhondali/remove-github-restrictions-message)或[此擴充套件](https://github.com/MohamadKh75/ShutHub)或[此Tampermonkey 腳本](https://gist.github.com/HirbodBehnam/2e079e187be0b1b6a6bcb734ed88474e)或[Stylish](https://userstyles.org/styles/173827/hide-github-warning)来隱藏這個黃色提示框!~~
+**更新:** 此警告訊息現在已經被隱藏了。
+
+![github-octocat](https://user-images.githubusercontent.com/16706911/61997137-7aa7df00-b0b2-11e9-97f1-f452855fe21c.png)
+
+## 介绍
+
+GitHub 曾經是一個對所有人都開放且自由的平台，但它現在卻決定限制伊朗用戶的使用、將之排除於開源生態。即使我们知道 GitHub 可能是在美國政府的施壓下做出此決定，但我們希望 GitHub 採取更尊重用戶的措施。
```
Comment (iyotetsuya) [inline README-TW.md]: `即使我们知道` -> `即使我們知道`

**Comment by pesehr** [comment]
@amazingandyyy 
Please resolve the conversations

File: README-TW.md (hunk 4 above)
Comment (amazingandyyy) [reply README-TW.md]: updated

File: README-TW.md (hunk 2 above)
Comment (amazingandyyy) [reply README-TW.md]: thanks!

File: README-TW.md (hunk 5 above)
Comment (amazingandyyy) [reply README-TW.md]: thanks!

**Pull request merged**